spark/src/test/java/org/opensearch/sql/spark/flint/operation/FlintIndexOpTest.java

```diff
@@ -33,54 +34,53 @@ public class FlintIndexOpTest {
   public void testApplyWithTransitioningStateFailure() {
     FlintIndexMetadata metadata = mock(FlintIndexMetadata.class);
     when(metadata.getLatestId()).thenReturn(Optional.of("latestId"));
-    FlintIndexStateModel fakeModel =
-        new FlintIndexStateModel(
-            FlintIndexState.ACTIVE,
-            metadata.getAppId(),
-            metadata.getJobId(),
-            "latestId",
-            "myS3",
-            System.currentTimeMillis(),
-            "",
-            SequenceNumbers.UNASSIGNED_SEQ_NO,
-            SequenceNumbers.UNASSIGNED_PRIMARY_TERM);
+    FlintIndexStateModel fakeModel = getFlintIndexStateModel(metadata);
     when(flintIndexStateModelService.getFlintIndexStateModel(eq("latestId"), any()))
         .thenReturn(Optional.of(fakeModel));
     when(flintIndexStateModelService.updateFlintIndexState(any(), any(), any()))
         .thenThrow(new RuntimeException("Transitioning state failed"));
     FlintIndexOp flintIndexOp =
         new TestFlintIndexOp(flintIndexStateModelService, "myS3", mockEmrServerlessClientFactory);
+
     IllegalStateException illegalStateException =
         Assertions.assertThrows(IllegalStateException.class, () -> flintIndexOp.apply(metadata));
+
     Assertions.assertEquals(
         "Moving to transition state:DELETING failed.", illegalStateException.getMessage());
   }
 
+  @NotNull
+  private static FlintIndexStateModel getFlintIndexStateModel(FlintIndexMetadata metadata) {
```

Comment:
nit: please move to beginning/end of class.. now it's between two test cases

Reply:
Fixed.